render.js

```diff
@@ -0,0 +1,46 @@
+const Renderer = function(){
```

Comment:
when you write code keep pressing alt + shift + f
this formats your code so everything is in the write place. It makes reading code a lot easier :)

if it is your first time pressing those keys you might get a popup in the bottom write. Click 'configure' and at the top of visual studio you will see a dropdown menu. Select the formatter you want to use. Let me know if you need help.